rust/src/index/vector/kmeans.rs

```diff
@@ -69,7 +70,8 @@ pub async fn train_kmeans(
             let s = array.slice(idx * dimension, dimension);
             builder.append_slice(as_primitive_array::<Float32Type>(s.as_ref()).values());
         }
-        builder.finish()
+        let d = builder.finish();
```

Comment:
don't need the extra variable?

Reply:
oh was to calculate the latency for a print. will revert this change.